pages/your_domain/structural_bioinformatics.md

```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
+* To extend molecular context, structure models can be visualized and analyzed along with respective volume maps and {% tool "uniprot" %} sequences using data aggregators such as {% tool "3dbionotes" %}, either if they have been published in PDB and EMDB or if not. A bunch of biochemical, biomedical and validation annotations would be mapped on the structure model coordinates and sequence to let the user better understand macromolecular function and interactions. Users can also use the [COVID-19 Structural Hub](https://3dbionotes.cnb.csic.es/ws/covid19), a dedicated summary view for all published SARS-CoV-2 structure models. 
+* As well as for computationally solved structures, model coordinates and metadata for experimentally solved structures are also preferably stored using the standard PDB archive format {% tool "pdbx-mmcif-format-and-tools" %} and the {% tool "pdbx-mmcif-modelcif-extension-dictionary" %}, respectively.
+* Data model and metadata standards for submitting data underpinning macromolecular models depend on the experimental method used. EMDB map distribution format description has broadly followed [CCP4](https://www.ccp4.ac.uk/html/maplib.html) and [MRC](https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/image-processing-software/) map format. Metadata is contained in a header file, an XML file that follows the XSD data model. EMPIAR data model schema consists of the main empiar.xsd XML schema file and additional requirements in empiar.sch in Schematron format (see [here](ftp://ftp.ebi.ac.uk/pub/databases/emtest/empiar/schema/) for more details). BMRB (meta)data distribution format is based on NMR-STAR, an extension of the Self-defining Text Archive and Retrieval (STAR) file format.
+* As image processing framework, users can operate with workflow managers FAIR compliant such as {% tool "scipion" %} to obtain macromolecular models using Electron Microscopy (3DEM). It integrates several software packages while taking care of formats and conversions and data submissions to public repositories such as EMDB and EMPIAR. It is also possible to deploy a cloud-compatible version of Scipion either in a single server or in a cluster with [ScipionCloud](https://scipion-em.github.io/docs/release-3.0.0/docs/developer/scipion-on-the-cloud.html#scipion-on-the-cloud). It can be also deployed in EOSC cloud infrastructures (see [here](https://marketplace.eosc-portal.eu/services/eosc.cnb-csic.scipioncloud?pv=search/all&q=scipioncloud) for details).
+* [ARIA](https://aria.services)  is a platform that projects and infrastructures can use to manage Access, from proposal submission through to reporting. It provides tools for facilities within a distributed infrastructure to manage their equipment.
+
+  If using ARIA to manage access to the facility, it will shortly be possible to link output data and metadata with proposals, publications and other outputs. This work is being funded by a number of EU funded grants.
```

Comment:
Why not add this feature to the previous paragraph? The sentence regarding funders is probably not useful for the reader (and I suppose all the tools mentioned on every page are funded somehow, and we do not want to write a similar sentence for each one).

Reply:
Done!